_posts/2024-03-09-neue-server-am-standort-muc.markdown

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title:  "Neue Server am Standort München seit 09.03.2024"
+date:   2024-03-09 20:00:00
+categories: freifunkmuc
+---
+
+**Neue Server am Standort München seit 09.03.2024**
+
+Am 09. März 2024 wurden an unserem Standort in Münchnen von unserm Team die alten Server gegen neue ausgetauscht.  
+Es stehen jetzt statt der insgesamt 32 Kerne in sehr alten Intels 256 Kerne mit Epycs zur Verfügung:  
+München läuft nun mit viel mehr Power! Auch unser Public DNS Dienst ist nun viel schneller!  
+Außerdem haben wir jetzt auch noch Peering mit Google.
+   
+Viel Spaß beim Nutzen :).
+   
+   
+![Neue Server](/assets/posts/2024-03-12-neue-server.jpeg) 
+
+![Neues Tempo](/assets/posts/2024-03-12-neues-tempo.jpeg)
+   
+Weitere Infos zu z.B. den Servern finden Sie bei uns im [Chat](https://chat.ffmuc.net) auf [Twitter/X](https://twitter.com/FreifunkMUC/status/1762382921451684233) oder [Mastodon](https://social.ffmuc.net/@freifunkMUC)
+   
+Wir bedanken uns bei ThomasKrennAG und EMC Home of Data für die Unterstützung!
```

Comment:
make ThomasKrennAG und EMC Home of Data clickbar link to:
https://www.thomas-krenn.com/
https://www.emc-homeofdata.de/

also update to "Thomas-Krenn.AG" and "EMC Home of Data GmbH"